packages/website/types.ts

```diff
@@ -0,0 +1,47 @@
+export interface Page {
+  id: string;
+  pagePath: string;
+  children: undefined;
+}
+
+export interface ExamplePage extends Page {
+  isolatedPath: string;
+}
+
+export type NestedExamples =
+  | { id: string; children: NestedExamples[] }
+  | ExamplePage;
+
+export type NestedPages = { id: string; children: NestedPages[] } | Page;
+
+export type Pages = Page[];
+
+type Maintainers = string | string[];
+
+type Repository = string | { type: string; url: string; directory?: string };
+
+type packageInfo2 = {
```

Comment:
should be capitalised. also is there any better way to name it besides Blank2?

Reply:
Yes, yes there is. Will fix